dcpquery-api.yml

```diff
@@ -8,8 +8,8 @@ info:
     <a href="https://www.humancellatlas.org/data-sharing">Data Coordination Platform</a> (DCP). Metadata from the
     <a href="https://github.com/HumanCellAtlas/data-store">DCP Data Store</a> are indexed and stored in an
     <a href="https://aws.amazon.com/rds/aurora/">AWS Aurora</a> <a href="https://www.postgresql.org/">PostgreSQL</a>
-    database.
-
+    database. For more user documentation, and example queries check out the
```

Comment:
Please use formal English and check the placement of punctuation:

"For more user documentation and example queries, see [Query Service documentation on Read the Docs](link)"